docs/android/v2/how-to-guides/extend-capabilities/noise-cancellation.mdx

```diff
@@ -14,8 +14,10 @@ implementation "live.100ms:hms-noise-cancellation-android:$hmsVersion"
 ```
 2. Toggle noise cancellation on in your application with `hmsSDK.setNoiseCancellationEnabled(true)` in your `onJoin` callback.
 
-> Note: Prebuilt also supports noise cancellation, to enable it add the import as above and ensure it's enabled from your prebuilt dashboard. From the [dashboard](https://dashboard.100ms.live/), select the template, go to "Customize Prebuilt" -> "Screens and Components" -> "Noise Cancellation State".
-> You will also need to toggle it in the [dashboard's](https://dashboard.100ms.live/)"Template" -> "Advanced Settings" -> "Noise Cancellation"
+<Note type='warning'>
+**IMPORTANT** <br/>
+Enable Noise Cancellation in the template configuration. Learn more on enabling this feature from [here](/get-started/v2/get-started/features/noise-cancellation#enabling-the-noise-cancellation)
```

Comment:
Sentence update:

Learn more about enabling this feature [here](/get-started/v2/get-started/features/noise-cancellation#enabling-the-noise-cancellation).